kube/src/lib.rs

```diff
@@ -317,10 +317,7 @@ mod test {
 
         let client = Client::try_default().await?;
         let api: Api<PodSimple> = Api::default_namespaced_with(client, &ar);
-        for p in api.list(&Default::default()).await? {
-            // run loop to cover ObjectList::iter
-            println!("found pod {} with containers: {:?}", p.name(), p.spec.containers);
-        }
+        let _ = api.list(&Default::default()).await?.iter().map(ResourceExt::name);
```

Comment:
Unused?

Reply:
yeah, it was doing a println before which also doesn't do much in the cluster, just trying to test the iterator thing on `ObjectList`

Reply:
i can't really assert anything about the pod here - because we don't know if the cluster has any

Reply:
added some other tests for the uncovered parts of ObjectList iterators and kube-core in another test case